gapic/templates/noxfile.py.j2

```diff
@@ -169,11 +169,4 @@ def blacken(session):
         *BLACK_PATHS,
     )
 
-
-@nox.session(python=DEFAULT_PYTHON_VERSION)
-def lint_setup_py(session):
```

Comment:
We're still doing the lint checks?

Reply:
We are still running `black` via the `lint` nox session which will fail on syntax errors: https://github.com/googleapis/gapic-generator-python/blob/bd81e52806c613bbf2a2805e7630aaa1ae8c98cc/gapic/templates/noxfile.py.j2#L158